scripts/taxonomize

```diff
@@ -0,0 +1,120 @@
+#! /usr/bin/env python
+
+"""
+Generates a VAST taxonomy from other tools
+
+Example usage:
+
+   taxonomize --splunk < model.json > splunk.yaml
+   taxonomize --sentinel > sentinel.yaml
```

Comment:
Where is the input coming from? It looks like it requires internet access. The difference in behavior compared to `--splunk` is not intuitive.

Either change the tool to work the same way in both cases or document the difference. Your call.

Reply:
I clarified this in a comment.